be/src/runtime/data_stream_recvr.h

```diff
@@ -160,7 +159,7 @@ class DataStreamRecvr {
     std::atomic<int> _num_buffered_bytes{0};
 
     // Memtracker for batches in the sender queue(s).
-    std::unique_ptr<MemTracker> _mem_tracker;
+    MemTracker* _mem_tracker = nullptr;
```

Comment:
why change this to raw pointer?

Reply:
There is no need to create a sub MemTracker here, use the MemTracker of instance directly